bgp-monitor-action-pack/bgp-monitor/script.py

```diff
@@ -55,6 +56,31 @@ def IsStatsDiffExpected(prevStats, currentStats, expectedDiff: int):
         err = "Missing change control device" if device is None \
             else f"device {device} is missing 'id'"
         raise ActionFailed(err)
+
+    # This block checks for the check_established flag, defaults to False
+    # If set to True, this block will check that all BGP peers are "Established"
+    # If any BGP peers are not Established, the CC will abort/fail
+    if checkEstablished:
+        bgpPeerState = ctx.runDeviceCmds(['enable', 'show ip bgp summary vrf all', 'show bgp evpn summary'])
```

Comment:
Maybe pull this out to a constant at the top of the file:
CHECK_PEERS_CMDS = ['enable', 'show ip bgp summary vrf all', 'show bgp evpn summary']